utilities/docker/ubuntu-xvfb/Dockerfile

```diff
@@ -0,0 +1,67 @@
+# MAINTAINER Dan Petre dan.petre@intel.com
```

Comment:
We don't specify contributor's name for each code. Please use the following.

```
# Open3D: www.open3d.org
# The MIT License (MIT)
# See license file or visit www.open3d.org for details
```

Github will eventually track contributors line by line.